src/themes/default/collections/menu.variables

```diff
@@ -119,6 +119,7 @@
 
 @labelTextMargin: 1em;
 @labelVerticalPadding: 0.3em;
+@labelVerticalPaddingSecondaryMenu: 1.2em;
```

Comment:
Thanks for taking a time to open this PR. 👍 

I think this looks very nice if labels are very narrow.
However, I am afraid that it overlay the parent item if the label has 3 or more characters,
![image](https://user-images.githubusercontent.com/127635/56456091-75940f00-63a2-11e9-933a-2f5e63e54fbf.png)

Another PR #669 looks like below, where I personally feels a gap is a bit longer.
![image](https://user-images.githubusercontent.com/127635/56456098-86dd1b80-63a2-11e9-8cce-52a1c0789588.png)

What if we [shorten a gap between label and item like this](https://jsfiddle.net/o2ps4ve3/)?
![image](https://user-images.githubusercontent.com/127635/56456226-572f1300-63a4-11e9-85f7-e4c1a9b6c4e1.png)

Reply:
I second @exoego . I also think 1.2em is too narrow for the mentioned reasons. However the suggestion from @exoego is still too narrow for me, because the top floating label will touch the icon and text of the adjacent item if the text gets too wide.
I suggest using `margin-top: 0`. See https://jsfiddle.net/mrupeckx/

Reply:
More detailed reason to use `margin-top:0` : The gap is identical (even more when using 0.05em) for top/bottom to the item, especially when the item has a usual label inside. See https://jsfiddle.net/mrupeckx/2/
![image](https://user-images.githubusercontent.com/18379884/56456898-179c0300-6373-11e9-8c3a-4beceeb2b6c8.png)

Reply:
Good idea. I didn't test it well enough to see these issues unfortunately

Reply:
Will you continue to work in this PR?

Reply:
As in: consider the suggestions and modify the PR? Or as in: are you still doing something because I'd like to wrap this up?

Reply:
Consider the suggestion as long as nobody else steps into the discussion 🙂

Reply:
Sorry, I don't know what to do now. Do I need to do something at the Changes requested  section?

Reply:
If you follow my suggestion, then you only need to change the code in your branch accordingly and push the changes. Then this PR gets updated automatically and I can approve it. 🙂

Reply:
Done :+1: